android/src/main/java/com/bitmovin/player/reactnative/BufferModule.kt

```diff
@@ -0,0 +1,73 @@
+package com.bitmovin.player.reactnative
+
+import com.bitmovin.player.api.buffer.BufferLevel
+import com.bitmovin.player.api.media.MediaType
+import com.bitmovin.player.reactnative.converter.JsonConverter
+import com.facebook.react.bridge.*
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.UIManagerModule
+
+private const val MODULE_NAME = "BufferModule"
+
+@ReactModule(name = MODULE_NAME)
+class BufferModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+    /**
+     * Collection of [BufferLevel] objects
+     * @param audio [BufferLevel] for [MediaType.Audio].
+     * @param video [BufferLevel] for [MediaType.Video].
+     */
+    data class RNBufferLevels(val audio: BufferLevel, val video: BufferLevel)
```

Comment:
We had the same case and discussion on another PR and we came to the conclusion to move these RN only types to the bottom of the file for Kotlin.

Reply:
Address in f57810589c0b93d2f4af52ba23e5e170e215580a

Reply:
This still seems to be inside of `BufferModule` type, I meant to move it outside of the `BufferModule` declaration, so outside of the last `}`

Reply:
Sorry, I misunderstood your previous comment.  
Address in 5c0d1c3e8e720e0df6df6a3eb47c330a7608cc82